### PR TITLE
refactor: remove unnecessary code

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

- Remove old, not used in component (from few PRs) code - I created this PR separately, because https://github.com/asyncapi/asyncapi-react/pull/301 has a lot changed files (with css and JS) and it's hard to review, this PR only have removed JS code.
- Update `umd` bundles from `@asyncapi/react-component/bundles/umd/...` to `@asyncapi/react-component/browser` to improve DX.
- Update docs for `umd` bundles - Readme.md and in NextJS tutorial.
- Update deps in root package.json (by this PR has about 20k Changed LOC 😅)

**Related issue(s)**
Part of https://github.com/asyncapi/shape-up-process/issues/83

### CODE REVIEW

File: library/package.json [hunk 1]
```diff
@@ -34,26 +34,27 @@
...
+    "build:styles": "npm run build:styles:dev && npm run build:styles:prod",
+    "build:styles:dev": "cross-env NODE_ENV=production postcss src/styles/default.css -o styles/default.css --verbose",
+    "build:styles:prod": "cross-env NODE_ENV=production MINIFY_STYLES=true postcss src/styles/default.css -o styles/default.min.css --verbose",
     "test": "jest --detectOpenHandles",
     "test:watch": "jest --detectOpenHandles --watch",
-    "prepare": "npm run build:prod",
-    "prepack": "cp ../README.md ./README.md && cp ../LICENSE ./LICENSE",
+    "prepare": "npm run build:dev",
+    "prepack": "npm run build:prod && cp ../README.md ./README.md && cp ../LICENSE ./LICENSE",
     "postpack": "rm -rf ./README.md && rm -rf ./LICENSE",
```
Comment (derberg) [inline library/package.json:58]: what is this for? why do we remove readme and license

Comment (magicmatatjahu) [reply library/package.json:58]: License and Readme are moved from root of project to library before publish, so by this library hasn't Readme in git, but we have it in npm https://www.npmjs.com/package/@asyncapi/react-component?activeTab=dependencies
After postpack we remove it (but these are copies).

Comment (derberg) [reply library/package.json:58]: perfect, thanks for explanation, I wish there was `package.yml` so we could have comments for future :(

Comment (magicmatatjahu) [reply library/package.json:58]: 🤦🏼 Due to this https://github.com/asyncapi/asyncapi-react/blob/master/package.json#L20 from this commit https://github.com/asyncapi/asyncapi-react/commit/a00fcc9cd98de3133397eac5d6a02ecb6163ebcd currently we haven't License and Readme on npm...

**Review by derberg — APPROVED** [review]

File: library/package.json (hunk 1 above)
Comment (magicmatatjahu) [reply library/package.json:58]: I will fix it the PR with bumping dependencies for "old" component.

**Pull request merged**